web/src/components/FormPersonnel.js

```diff
@@ -0,0 +1,152 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Button } from 'rebass';
+import { Field, FieldArray, reduxForm } from 'redux-form';
+
+import { Input } from './Inputs';
+
+const PersonnelList = ({ fields, meta: { error, submitFailed } }) => (
+  <div>
+    <div>{submitFailed && error && <span>{error}</span>}</div>
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Job title</th>
+          <th>Job description</th>
+          <th>2019 compensation</th>
+          <th>2019 percentage of time</th>
+          <th>2019 cost to activity</th>
+          <th>2020 percentage of time</th>
+          <th>2020 cost to activity</th>
+          <th />
+        </tr>
+      </thead>
+      <tbody>
+        {fields.map((person, idx) => (
+          <tr key={person}>
+            <td>
+              <Field
+                hideLabel
+                label="personnel name"
+                name={`${person}.name`}
+                component={Input}
+              />
+            </td>
+            <td>
+              <Field
+                hideLabel
+                label="job title"
+                name={`${person}.jobTitle`}
+                component={Input}
+              />
+            </td>
+            <td>
+              <Field
+                hideLabel
+                label="job description"
+                name={`${person}.jobDescription`}
+                component={Input}
+              />
+            </td>
+            <td>
+              <Field
+                hideLabel
+                label="2019 compensation"
+                name={`${person}.nextCompensation`}
+                component={Input}
+                parse={value => Number(value)}
```

Comment:
can also do `+value` to convert to value -- more terse than `Number(value)` though also less clear (i have no pref except for consistency)